src/components/twilio-voice-dialer/public/twilio-voice-dialer.js

```diff
@@ -11,12 +12,37 @@ class TwilioVoiceDialer extends HTMLElement {
     this.attachShadow({ mode: 'open' });
   }
 
+  attributeChangedCallback() {
+    this.#render();
+  }
+
+  connectedCallback() {
+    this.shadowRoot
+      .querySelector('#call')
+      .addEventListener('click', () => this.#handleCall());
+    this.shadowRoot
+      .querySelector('#hangup')
+      .addEventListener('click', () => this.#handleHangup());
+  }
+
   disconnectedCallback() {
     this.#device.destroy();
   }
 
-  attributeChangedCallback() {
-    this.#render();
+  async #handleCall() {
+    const recipient = this.shadowRoot.querySelector('#recipient').value;
+    if (recipient) {
+      this.#call = await this.#device.connect({ params: { recipient } });
+      this.#setupCallHandlers(this.#call);
+      this.#setStatus('inprogress');
+    } else {
+      console.log('recipient is required to make a call');
+    }
+  }
+
+  #handleHangup() {
+    this.#setStatus('idle');
```

Comment:
I just realized we're already setting this on disconnect (line 81). Can we remove this here?